ui/app/components/namespace-picker.ts

```diff
@@ -53,10 +55,7 @@ export default class NamespacePicker extends Component {
   }
 
   private matchesPath(option: NamespaceOption, currentPath: string): boolean {
-    // TODO: Revisit. A hardcoded check for "path" & "/path" seems hacky, but it fixes a breaking test:
-    //  "Acceptance | Enterprise | namespaces: it shows nested namespaces if you log in with a namespace starting with a /"
-    //  My assumption is that namespace shouldn't start with a "/", but is this a HVD thing? or is the test outdated?
-    return option?.path === currentPath || `/${option?.path}` === currentPath;
+    return option?.path === currentPath;
```

Comment:
🗒️ Note: traced this back to [this PR](https://github.com/hashicorp/vault/pull/7186). The test in question was added to verify that the namespace picker works whether the user logs in with '/namespace' or 'namespace'. The namespace service strips out any prefixed '/', so `option?.path` will not have a proceeding '/'. We still have an enterprise test to ensure that the namespace picker works whether the user logs in w/ a '/' prefixed namespace or not.